tests/Kernel/Paragraphs/ParagraphsTest.php

```diff
@@ -194,64 +188,45 @@ public function testListItem(): void {
     $html = $this->renderParagraph($paragraph);
     $crawler = new Crawler($html);
 
-    $this->assertCount(1, $crawler->filter('.ecl-list-item.ecl-list-item--highlight'));
-    $this->assertEquals('Item title', trim($crawler->filter('.ecl-list-item__title')->text()));
-    // The description should not be rendered in this variant.
-    $this->assertCount(0, $crawler->filter('.ecl-list-item__detail'));
-    // Neither the date.
-    $this->assertCount(0, $crawler->filter('.ecl-date-block__week-day'));
-    $this->assertCount(0, $crawler->filter('.ecl-date-block__day'));
-    $this->assertCount(0, $crawler->filter('.ecl-date-block__month'));
-    // Neither the metas.
-    $this->assertCount(0, $crawler->filter('.ecl-meta__item'));
-
-    $link_element = $crawler->filter('.ecl-list-item__link');
+    $this->assertEquals('Item title', trim($crawler->filter('article.ecl-card header.ecl-card__header h1.ecl-card__title')->text()));
+
+    $link_element = $crawler->filter('article.ecl-card header.ecl-card__header h1.ecl-card__title a.ecl-link');
     $this->assertCount(1, $link_element);
     $this->assertEquals('http://www.example.com/', $link_element->attr('href'));
 
-    $image_element = $crawler->filter('.ecl-list-item__primary img.ecl-image');
+    $image_element = $crawler->filter('article.ecl-card header.ecl-card__header div.ecl-card__image');
     $this->assertCount(1, $image_element);
-    $this->assertEquals(
+    $this->assertContains(
       file_url_transform_relative(file_create_url($image->getFileUri())),
-      $image_element->attr('src')
+      $image_element->attr('style')
     );
-    $this->assertEquals('Druplicon', $image_element->attr('alt'));
-
+    // @todo will be fixed on OPENEUROPA-2123.
+    // $this->assertEquals('Druplicon', $image_element->attr('alt'));
     // Change the variant to thumbnail primary.
     $paragraph->get('oe_paragraphs_variant')->setValue('thumbnail_primary');
     $paragraph->save();
 
     $html = $this->renderParagraph($paragraph);
     $crawler = new Crawler($html);
 
-    $this->assertCount(1, $crawler->filter('.ecl-list-item.ecl-list-item--thumbnail'));
```

Comment:
This asserts that we don't show both images, primary and secondary, and it should be kept.

Reply:
Tested during link of the image.